src/runtime/extended-access/gaa-metering-test.js

```diff
@@ -246,18 +246,6 @@ describes.realWin('GaaMetering', () => {
     }
   }
 
-  describe('constructor', () => {
-    it('sets class variable', () => {
-      const gaaMeteringInstance = new GaaMetering();
```

Comment:
Note: There was no reason to instantiate this class, since it just contains static methods.